src/main/java/lab6/ConversionPanel.java

```diff
@@ -0,0 +1,75 @@
+package lab6;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import java.util.function.DoubleUnaryOperator;
+
+/**
+ * Custom component that handles a single type of conversion
+ */
+public class ConversionPanel extends VBox {
+    private TextField fromField;
+    private TextField toField;
+
+    public ConversionPanel(String fromUnit, String toUnit,
+                           DoubleUnaryOperator convertTo,
+                           DoubleUnaryOperator convertFrom) {
+        setSpacing(10);
+        setAlignment(Pos.CENTER);
```

Comment:
Good idea to break this out into a class that can be reused to represent the different kinds of conversions the app can do